src/vite/templates.ts

```diff
@@ -138,7 +138,7 @@ function mergeProperty (storeModule, moduleData, property) {
 export const clientConfigTemplate = {
   filename: 'nitro.client.mjs',
   getContents: () => `
-export const useRuntimeConfig = () => window?.__NUXT__?.config || {}
+export const useRuntimeConfig = function(){return window && window.__NUXT__ && window.__NUXT__.config || {}}
```

Comment:
Unlike the other two files, this code will be processed by rollup/vite. Shouldn't features like optional chaining or arrow functions be compiled out if the right target is specified?

Reply:
Additionally, browser support for arrow functions is below modern baseline (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).